### PR TITLE
Feature 1

### PR DESCRIPTION
**Pull request by fithamlak**
hello microverse task to check all the concepts on linter, git-flow, and professional Repo

### CODE REVIEW

**Review by Alfredbis29 — CHANGES_REQUESTED** [review]
Hi @fithamlak  , 

Great work so far 👏 
You've done well implementing some requirements on this project, but there are still some which aren't addressed yet.


 ## Highlights


 ## Required Changes ♻️

- [ ] Please ensure that linters are appropriately configured so that they can run to verify that your pull request is free of errors. I advise you to read this after taking a moment to reflect.[ lesson](https://github.com/microverseinc/linters-config/blob/master/html-css) to understand it clearly and follow the guideline to set it correctly.

 You should consider to run this command:

`To install webhint depencies: npm install --save-dev hint@7.x
Stylelint: npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x`

- [ ] Kindly make sure to document your README professionally. Please use the [Microverse README template](https://github.com/microverseinc/readme-template) and customize it. :smile:
 
- [ ] Add a descriptive PR  as  well , kindly describe all you did in this project :point_down: 

 To add a PR description on your project , Click on the `three dot` , then `edit` there you will be able to add your description  
description


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1,2 +1,34 @@
...
+
+**[Hell Microverse]** is a a simple task to master all of the tools and best practices I learned about linter, githubflow and documenting readme professionally.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@fithamlak](https://github.com/fithamlak)
```
Comment (Nyame-Wolf) [inline README.md]: Great work so far with the requested changes. however as requested by the previous reviewer, kindly use the microverse [readme template](). Its a great effort you have some of the elements the template has, but to make your project better and be ablt to showcase your work better, I suggest you copy the template as it is and edit it to fit this project

To copy, click on the copy button on the template as shown below by below arrow:
Screenshot 2022-11-30 at 16 40 02

**Review by Nyame-Wolf — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle: 

Hi there @fithamlak ! , 


Great job introducing the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some issues that still need to be addressed for your project to shine :sparkles:.

## Highlights


- Linters working as expected
- Fulfilled code requirememnts



Congrats on your work so far! :clap:  


 ## Required Changes ♻️

 There are some issues that you still need to work on, but you are nearly there! 
_For more information, check the comments under the review :arrow_down:_


## Optional suggestions :ballot_box_with_check: 

_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can improve the quality of your code :100:_

Happy coding and keep it up! :raised_hands: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
  Please, remember to tag me **@Nyame-Wolf**  in your question so I can receive the notification.

  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

  _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1,2 +1,34 @@
...
+
+  <h3><b>Description abou my hello microverse project</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
```
Comment (DuaneDave) [inline README.md]: Good job adding the required read me, but you have deleted key section that would make for a professional read me and I recommend adding them and filling them up with your details.

see below...


readmeeeee

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @fithamlak 👋 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✅ GitHub flow used correctly


 ## Required Changes ♻️
- As requested by the first reviewer, please add a descriptive pull request summary as this makes it easier for someone to know the tasks you have accomplished

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by iambenkis — APPROVED** [review]
### Hi @fithamlak  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


cr-approve


## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Optional suggestions

**_Every comment with the [**OPTIONAL**] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**